### PR TITLE
Add Xiaomi docs for platforms

### PR DESCRIPTION
**Pull request by fabaff**
**Description:**
Add Xiaomi docs for platforms. See #3028 for the initial PR.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

**Comment by fabaff** [comment]
The binary sensor needs to be merged with #3029.

File: source/_components/binary_sensor.xiaomi.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+description: "Instructions how to setup the Xiaomi binary sensors within Home Assistant."
+date: 2017-07-21 16:34
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi.png
+ha_category: Binary Sensor
+ha_release: "0.50"
+ha_iot_class: "Cloud Polling"
```
Comment (syssi) [inline source/_components/binary_sensor.xiaomi.markdown]: It think "Cloud Polling" isn't correct. The communication between HA and the xiaomi gateway is multicast/udp at the local network. The multicast broadcast pushs new events. Polling is just used to fix some issues.

Comment (fabaff) [inline source/_components/binary_sensor.xiaomi.markdown]: Thanks

**Comment by fabaff** [comment]
Thanks @syssi for providing all those automations.

File: source/_components/binary_sensor.xiaomi.markdown (hunk 1 above)
Comment (syssi) [inline source/_components/binary_sensor.xiaomi.markdown]: I would prefer "Local Push".

**Pull request merged**